src/revertex/generators/borehole.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import annotations
+
+import logging
+from collections.abc import Mapping
+
+import legendhpges
+import numpy as np
+from numpy.typing import ArrayLike, NDArray
+
+from revertex import core, utils
+
+log = logging.getLogger(__name__)
+
+
+def generate_hpge_borehole_points(
+    n_tot: int,
+    seed: int | None = None,
+    *,
```

Comment:
```suggestion
    *,
    seed: int | None = None,
```
i think this should be kwonly too, otherwise can be confused with n_tot